src/docs/product/accounts/choose-your-data-center/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Choose Your Data Center (US or EU)
+sidebar_order: 30
+description: "Choose where to store your data in the US or the EU."
+---
+
+Sentry supports data storage in both the US and the EU.
+
+You can choose where to store your data when you’re setting up your Sentry account by selecting from the dropdown menu under "Data Storage" in the "Create a New Organization" section.
+
+Your selection determines your data storage location only. Sentry will continue to access and process your data in accordance with your agreement with Sentry for the Sentry service and our [Privacy Policy](https://sentry.io/privacy/).
+
+<Note>
+  Once selected, your data storage location can’t be changed. The only way to
+  switch regions is by creating a new account.
+</Note>
+
+The table below shows the location of the data center where your data will be stored at rest, based on your selection.
+
+| **Data Center**                | **Location**       |
+| ------------------------------ | ------------------ |
+| United States of America (USA) | Iowa, USA          |
+| European Union (EU)            | Frankfurt, Germany |
```

Comment:
```suggestion
The table below shows the location where your data will be stored at rest, based on your selection.

| **Data Storage Location**      | **Physical Location** |
| ------------------------------ | --------------------- |
| United States of America (USA) | Iowa, USA             |
| European Union (EU)            | Frankfurt, Germany    |
```

Repeating "Location" twice in the headers feels a bit weird to me still but I couldn't come up with any better naming

Reply:
I think it's ok :)